locale/manageiq.pot

```diff
@@ -16,7 +16,7 @@ msgstr ""
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"Plural-Forms: nplurals=INTEGER; plural=EXPRESSION;\n"
+"Plural-Forms: nplurals=2; plural=n != 1;\n"
```

Comment:
I don't think that the template file should have plural definitions specific to the English language - as this template file is used by translators for all languages.

Reply:
> I don't think that the template file should have plural definitions specific to the English language - as this template file is used by translators for all languages.

that was already there before previous commit.

Reply:
correction, it was only there in en/manageiq.po file so i will revert this change